configs/metrics/kde.yaml

```diff
@@ -0,0 +1,51 @@
+baskerville: True
+metrics:
+  kde_umap_kl_approx:
+    class: kde
+    arguments:
+      embedding_name: inception_umap
+      embedding_kwargs:
+        batch_size: 16
+        device: cpu
+        n_components: 2
+        random_seed: 42
```

Comment:
`embedding_kwargs` for Baskerville should be `device="cuda"` (and below for other kde metrics using inception)

Reply:
Good spot! Have updated.